specs/rollup-node-p2p.md

```diff
@@ -258,28 +259,37 @@ Pre-Canyon/Shanghai blocks are broadcast on `/optimism/<chainId>/0/blocks`.
 
 Post-Canyon/Shanghai blocks are broadcast on `/optimism/<chainId>/1/blocks`.
```

Comment:
Style question: do we want to change this to `Post-Canyon/Shanghai, but Pre-Ecotone`? Another "clarity versus over-verbosity" balancing act I'm wondering about.

Reply:
I think that's a good call, let me update these topics to list which hardforks broadcast on them